pkg/services/serviceaccounts/manager/service.go

```diff
@@ -0,0 +1,50 @@
+package manager
+
+import (
+	"context"
+
+	"github.com/grafana/grafana/pkg/api/routing"
+	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/services/accesscontrol"
+	"github.com/grafana/grafana/pkg/services/serviceaccounts/api"
+	"github.com/grafana/grafana/pkg/services/serviceaccounts/database"
+	"github.com/grafana/grafana/pkg/services/sqlstore"
+	"github.com/grafana/grafana/pkg/setting"
+)
+
+type ServiceAccountsService struct {
+	store *database.ServiceAccountsStoreImpl
+	cfg   *setting.Cfg
+	log   log.Logger
+}
+
+func ProvideServiceAccountsService(
+	cfg *setting.Cfg,
+	store *sqlstore.SQLStore,
+	ac accesscontrol.AccessControl,
+	routeRegister routing.RouteRegister,
+) (*ServiceAccountsService, error) {
+	s := &ServiceAccountsService{
+		cfg:   cfg,
+		store: database.NewServiceAccountsStore(store),
+		log:   log.New("serviceaccounts"),
+	}
+	if err := ac.DeclareFixedRoles(role); err != nil {
+		return nil, err
+	}
+	if cfg.FeatureToggles["service-accounts"] {
+		serviceaccountsAPI := api.NewServiceAccountsAPI(s, ac, routeRegister)
+		serviceaccountsAPI.RegisterAPIEndpoints()
+	} else {
+		s.log.Debug("FeatureToggle service-accounts not found")
```

Comment:
Will this run every time grafana starts without the service-accounts toggle, or is this an error condition?